cudamapper/src/utils.cpp

```diff
@@ -115,6 +121,193 @@ void print_paf(const std::vector<Overlap>& overlaps,
     }
 }
 
+#ifdef GW_BUILD_HTSLIB
+static inline void encode_cigar(bam1_t* alignment, const std::string& cigar)
```

Comment:
Most of these functions could be put in an anonymous namespace as they are only used in this file

Reply:
done. moved all internal functions to anonymous namespace